<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSettingsWidget</class>
 <widget class="QWidget" name="GameSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="1" column="1">
      <widget class="QComboBox" name="m_variantCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_variantLabel">
       <property name="text">
        <string>Variant:</string>
       </property>
       <property name="buddy">
        <cstring>m_variantCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="m_timeControlBtn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_timeControlLabel">
       <property name="text">
        <string>Time Control:</string>
       </property>
       <property name="buddy">
        <cstring>m_timeControlBtn</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_ponderingCheck">
     <property name="text">
      <string>Thinking on opponent's time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Draw adjudication</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="m_drawMoveNumberLabel">
        <property name="text">
         <string>Move number:</string>
        </property>
        <property name="buddy">
         <cstring>m_drawMoveNumberSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_drawMoveNumberSpin">
        <property name="toolTip">
         <string>The game can be adjudicated as a draw after this many full moves have been played. A value of 0 disables draw adjudication.</string>
        </property>
        <property name="specialValueText">
         <string>Off</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_drawMoveCountLabel">
        <property name="text">
         <string>Move count:</string>
        </property>
        <property name="buddy">
         <cstring>m_drawMoveCountSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="m_drawMoveCountSpin">
        <property name="toolTip">
         <string>The scores of both players need to be within the chosen threshold for this many consecutive moves.</string>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_drawScoreLabel">
        <property name="text">
         <string>Score:</string>
        </property>
        <property name="buddy">
         <cstring>m_drawScoreSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="m_drawScoreSpin">
        <property name="toolTip">
         <string>Both players' scores must be within this many centipawns from zero.</string>
        </property>
        <property name="suffix">
         <string> cp</string>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
